src/quo2/reanimated.cljs

```diff
@@ -3,22 +3,31 @@
             [reagent.core :as reagent]
             [clojure.string :as string]
             ["react-native-reanimated" :default reanimated
-             :refer (useSharedValue useAnimatedStyle withTiming withDelay Easing Keyframe)]))
+             :refer (useSharedValue useAnimatedStyle useAnimatedGestureHandler withTiming withDelay withSpring Easing Keyframe runOnUI runOnJS)]))
+
+(def reanimated-js (js/require "../src/js/reanimated.jsx"))
 
 ;; Animated Components
 (def create-animated-component (comp reagent/adapt-react-class (.-createAnimatedComponent reanimated)))
 
-(def view (reagent/adapt-react-class (.-View reanimated)))
-(def image (reagent/adapt-react-class (.-Image reanimated)))
+;; Components
+
+(def image (reagent/adapt-react-class (.-Image ^js reanimated)))
 (def touchable-opacity (create-animated-component (.-TouchableOpacity ^js rn)))
+(def scroll-view (reagent/adapt-react-class (.-ScrollView ^js reanimated)))
+
+(def view (reagent/adapt-react-class (.-ReanimatedView ^js reanimated-js)))
```

Comment:
could you elaborate please

Reply:
I created a custom wrapper component for `Reanimated.Text`, so we can directly integrate it into the status `text/text` component. In order to use animated text, we just have to pass an additional `:animated-style` prop, and it will automatically switch to the animated component instead of the default RN one

Reply:
I am still confused as to how this exported component will work differently from the component which we imported in cljs. Please can you explain with an example where it will work, but other will not?